Mathlib/Data/Matroid/Circuit.lean

```diff
@@ -319,4 +336,129 @@ lemma ext_iff_circuit {M₁ M₂ : Matroid α} :
     M₁ = M₂ ↔ M₁.E = M₂.E ∧ ∀ C, M₁.Circuit C ↔ M₂.Circuit C :=
   ⟨fun h ↦ by simp [h], fun h ↦ ext_circuit h.1 fun C hC ↦ h.2 (C := C)⟩
 
+section Elimination
+
+/-! ### Circuit Elimination -/
+
+variable {ι : Type*} (x : ι → α) (Is Cs : ι → Set α) {z : α} {C₁ C₂ : Set α}
```

Comment:
I'd rather not see explicit arguments in `variable` hiding from the actual declarations.
Also the names `Is Cs` are not idiomatic for functions.